src/bibtex/all.bib

```diff
@@ -402,15 +402,16 @@ @article{turing:1936
   year={1936}
 }
 
-@article{Kucukelbir:2015,
-  title = {Automatic Variational Inference in {S}tan},
-  author = {Alp Kucukelbir and Rajesh Ranganath and Andrew Gelman
-  and David M. Blei},
-  year = {2015},
-  journal = {arXiv},
-  volume = {1506.03431},
-  url = {http://arxiv.org/abs/1506.03431}
+                  
+@article{kucukelbir:2015,
+  title={Automatic variational inference in Stan},
+  author={Kucukelbir, Alp and Ranganath, Rajesh and Gelman, Andrew and Blei, David},
+  journal={Advances in neural information processing systems},
+  volume={28},
+  year={2015},
```

Comment:
The best reference here is JMLR from 2017:  https://www.jmlr.org/papers/volume18/16-107/16-107.pdf